doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -11,15 +11,29 @@ Param (
     $DocGenDir
 )
 
-Write-Verbose "Create variables for identifying package location and package safe names"
+Write-Verbose "Name Reccuring paths with variable names"
 $PackageLocation = "${ServiceDirectory}/${ArtifactName}"
+$FrameworkDir = "${BinDirectory}/${ArtifactName}/dll-docs"
```

Comment:
While this might work I think it is likely better to always use ArtifactDirectoryName and if it isn't set then set it to ArtifactName here. That way we don't have to maintain another set of paths under the cognitiveservices case below.